Makefile

```diff
@@ -2,4 +2,15 @@ run:
 	ENV=dev go run .
 
 run-compose:
-	docker-compose up -d
\ No newline at end of file
+	docker-compose up -d
+
+start-dev-db-linux:
+	docker start mysql-dev || docker run --name mysql-dev -d \
+        -p 3306:3306 \
+        -e MYSQL_ROOT_PASSWORD=P@ssw0rd \
+        -e MYSQL_DATABASE=garnbarn \
+        --restart unless-stopped \
+        mysql:8.0.32
```

Comment:
I already start the db in the `docker-compose`. So is this one necessary?

Reply:
Not quite. I did it because I can't run docker-compose at first, but I can run it right now. So I also think we don't need this part anymore

Reply:
I just tried to connect phpmyadmin to the database but it is on a different network due to docker compose. Do you think we need phpmyadmin? If not, I will remove both of them

Reply:
It is okay to have phpmyadmin. You may move the command into the `docker-compose` instead of a stand-alone command.